### ISSUE TITLE
[Bug] Keyboard Activation for Buttons dont work with

### ISSUE THREAD

**Issue by EnsnerT** [EnsnerT/DailyLog#11]
> [Bug] : Accessibility on Buttons `.time` does not work with "Enter" nor "Space" key! 

 _Originally posted by @EnsnerT in [#5](https://github.com/EnsnerT/DailyLog/issues/5#issuecomment-4032346843)_

Issue lies on `API.ContextMenuEvent` not handling the Event 'click' not 'keydown'.
